android/build.gradle

```
@@ -31,6 +35,6 @@ repositories {
 }
 
 dependencies {
-    compile 'com.facebook.react:react-native:+'
+    compile 'com.facebook.react:react-native:${safeExtGet('reactNativeVersion', '+')}'
```

There's a syntax error here resulting in a build failure:

```
* What went wrong:
A problem occurred evaluating project ':react-native-proximity'.
> Could not find method reactNativeVersion() for arguments [, )}] on DefaultExternalModuleDependency{group='com.facebook.react', name='react-native', version='${safeExtGet(', configuration='default'} of type org.gradle.api.internal.artifacts.dependencies.DefaultExternalModuleDependency.
```

```suggestion
    compile "com.facebook.react:react-native:${safeExtGet('reactNativeVersion', '+')}"
```